osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -73,47 +70,30 @@ private void load(OsuConfigManager config, [CanBeNull] SkinEditorOverlay skinEdi
                 new ExportSkinButton(),
                 new DeleteSkinButton(),
             };
-
-            config.BindWith(OsuSetting.Skin, configBindable);
         }
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            skinDropdown.Current = dropdownBindable;
+            skinDropdown.Current = skins.CurrentSkinInfo;
 
             realmSubscription = realm.RegisterForNotifications(_ => realm.Realm.All<SkinInfo>()
```

Comment:
It's already specified in BDL:
```suggestion
            realmSubscription = realm.RegisterForNotifications(_ => realm.Realm.All<SkinInfo>()
```